Jenkinsfile

```diff
@@ -0,0 +1,243 @@
+                dir(reframeDir) {
+                    checkout scm
+                    sh("""${loginBash}
+                          ${moduleDefinition}
+                          cp reframe/settings.py ${publicSettings}
+                          cp ${cscsSettings} reframe/settings.py
+                          bash ${reframeDir}/${bashScript} -f ${reframeDir} -i ''""")
+                }
+            }
+        }
+    }
+
+    try {
+        parallel builds
+        currentBuild.result = "SUCCESS"
+    } catch(err) {
+        if (err.toString().contains('exit code 143')) {
+            currentBuild.result = "ABORTED"
+            println "The Unittest was cancelled. Aborting....."
+        }
+        else if (err.toString().contains('Queue task was cancelled')) {
+            currentBuild.result = "ABORTED"
+            println "The Queue task was cancelled. Aborting....."
+        }
+        else {
+            currentBuild.result = "FAILURE"
+            println "The Unittest failed. Exiting....."
+        }
+    }
+}
+
+stage('Public Test') {
+    if (currentBuild.result != 'SUCCESS') {
+        println 'Not executing "Public Test" Stage'
+        return
+    }
+    else {
+        try {
+            node('dom') {
+                def scratch = sh(returnStdout: true,
+                             script: """${loginBash}
+                                        echo \$SCRATCH""").trim()
+                def reframeDir = "${scratch}/${dirPrefix}-dom-${shortCommit}"
+                dir(reframeDir) {
+                    sh("""${loginBash}
+                          bash ${reframeDir}/$bashScript -f ${reframeDir} -i '' -p""")
+                }
+            }
+            currentBuild.result = "SUCCESS"
+        } catch(err) {
+            if (err.toString().contains('exit code 143')) {
+                currentBuild.result = "ABORTED"
+                println "The Public Test was cancelled. Aborting....."
+            }
+            else if (err.toString().contains('Queue task was cancelled')) {
+                currentBuild.result = "ABORTED"
+                println "The Queue task was cancelled. Aborting....."
+            }
+            else {
+                currentBuild.result = "FAILURE"
+                println "The Public Test failed. Exiting....."
+            }
+        }
+    }
+}
+
+builds = [:]
+stage('Tutorial Check') {
+    if (currentBuild.result != 'SUCCESS') {
+        println 'Not executing "Tutorial Check" Stage'
+        return
+    }
+    else {
+        try {
+            node('daint') {
+                def scratch = sh(returnStdout: true,
+                                 script: """${loginBash}
+                                            echo \$SCRATCH""").trim()
+                def reframeDir = "${scratch}/${dirPrefix}-daint-${shortCommit}"
+                dir(reframeDir) {
+                    sh("""${loginBash}
+                          bash ${reframeDir}/${bashScript} -f ${reframeDir} -i '' -t""")
```

Comment:
I think you can just use `-t ''` here instead of `-t ""`

Reply:
The `"""` is the way to close the string argument (groovy syntax for multiline strings).

Reply:
Oh, yes! I was confused!